.gitignore

```diff
@@ -10,4 +10,6 @@ tmp-zip
 # Ignore Node.js technologies build and cache directories
 node_modules/
 .cache/
-.bundle/
\ No newline at end of file
+.bundle/
+jupyter-pandas.zip
```

Comment:
Why ? it's just some local files it's it ?

Reply:
you're right

Reply:
ALWAYS !